src/main/java/net/minecraftforge/common/config/Configuration.java

```diff
@@ -80,9 +89,16 @@
 
     public Configuration(){}
 
+    @Deprecated
+    public Configuration(CharSource staticSource, CharSink staticSink){
```

Comment:
I don't see what the `PARENT` has to do with the file. `PARENT` is just used to save all configs into one global config file. This has nothing to do with whether you use files or the source/sink abstraction.